recipe/meta.yaml

```diff
@@ -0,0 +1,68 @@
+{% set name = "mcl" %}
+{% set version = "1.86.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/herumi/{{ name}}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: a3ab1cd01a6419f46210fd5874e795e37a4565fa305e0302301b31ffed208872
+  patches:
+    - patches/0001-Don-t-install-static-library.patch
+    - patches/0002-Move-Windows-shared-library-in-bin.patch
+
+build:
+  skip: True  # [s390x]
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - clang    # [linux and aarch64]
```

Comment:
I see this is the same as the `build.sh`. Can you comment why this is necessary?

Reply:
yes, it is clang on linux aarch64, see 
https://github.com/herumi/mcl/blob/79774aa88da73d58d5fd4da209cc5373770f8009/CMakeLists.txt#L150-L166

Reply:
Please add a comment in the recipe.